### ISSUE TITLE
Toggle Usage and Style documentation needs some changes

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon-website#3065]
A review of the toggle's [Usage](https://ibm.box.com/s/h3xlz2ofl9yp7i0tio6twjzzg61oaglj) and [Style](https://ibm.box.com/s/lujqq2w3b14m95ux59o85ay3bhq1zdjb) pages was carried out as part of an accessibility review.  Note that there are comments from both Jess Lin and myself (and I sometimes respond to something she said).

Key items to consider:
- on the Style page, an 'off' toggle is shown with a darker switch circle, which makes the entire construct potentially fail non-text contrast (since the contrast between the outline and the circle is only 2.5:1). It appears this is an anomaly (does not appear on the React components page for example), but the toggle button presentation on all 4 themes (white, grey 10, etc) should be carried out to ensure that the color of the circle is the same as the page background AND that the outline of the toggle contrasts at least 3:1 against this background.
<img width="270" alt="image" src="https://user-images.githubusercontent.com/14298245/181919906-c69c1aed-4e0e-4087-9e2f-35274019afa9.png">
^ bad example
<img width="250" alt="image" src="https://user-images.githubusercontent.com/14298245/181919956-c3881234-0a73-4d14-a451-6110f7fe34dc.png">
^ passing example

- the Usage page lacks an Anatomy section, which would go a long way to answering some questions we had about how a Heading would be used in a toggle (there are no examples). It may also help align terms on the usage and style pages.
- the usage page does not list all the variants. There are at least 2, possibly 3: the default, a small WITH on/off text, and a small without on/off text. It's worth making it clear that the on/off text is only optional on the Small variant, not the default

This document should be opened in Adobe Acrobat Reader (not in a browser) so that all the comments and replies can be reviewed.

**Comment by sstrubberg** [comment]
Needs new Usage template treatment.

**Comment by thyhmdo** [comment]
Looks like the Toggle doc has been updated, most of issues were already addressed. Designers can double check/consider if these feedback is outdated

- [ ] Adding an Anatomy image
- [ ] Listing all variants (2- Default and Small)
- [ ] Improving the color contrast of the Off toggle\
- [ ] Updating some issues in the Content section
- [ ] Considering the Adding information to Carbon Data table component https://github.ibm.com/IBMa/E2E/issues/4761

**Comment by thyhmdo** [comment]
Thank you! @mbgower
